sdk/search/azure-search-documents/azure/search/documents/indexes/_search_index_client.py

```diff
@@ -477,3 +477,162 @@ def get_service_statistics(self, **kwargs):
         result = self._client.get_service_statistics(**kwargs)
         return result.as_dict()
+
+    @distributed_trace
+    def list_aliases(self, **kwargs: Any) -> ItemPaged[SearchAlias]:
+        # type: (**Any) -> ItemPaged[SearchAlias]
+        """List the aliases in an Azure Search service.
+
+        :keyword select: Selects which top-level properties of the skillsets to retrieve. Specified as a
+         list of JSON property names, or '*' for all properties. The default is all
+         properties.
+        :paramtype select: list[str]
+        :return: List of Aliases
+        :rtype: ~azure.core.paging.ItemPaged[~azure.search.documents.indexes.models.SearchAlias]
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        """
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+        if kwargs.get('select', None):
+            kwargs['select'] = ','.join(kwargs['select'])
+        # pylint:disable=protected-access
+        return self._client.aliases.list(
+            cls=lambda objs: [x for x in objs], **kwargs
+        )
+
+    @distributed_trace
+    def list_alias_names(self, **kwargs: Any) -> ItemPaged[str]:
+        # type: (**Any) -> ItemPaged[str]
+        """List the alias names in an Azure Search service.
+
+        :return: List of alias names
+        :rtype: ~azure.core.paging.ItemPaged[str]
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        """
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+
+        return self._client.aliases.list(
+            cls=lambda objs: [x.name for x in objs], **kwargs
+        )
+
+    @distributed_trace
+    def get_alias(self, name: str, **kwargs: Any) -> SearchAlias:
+        # type: (str, **Any) -> SearchAlias
+        """
+
+        :param name: The name of the alias to retrieve.
+        :type name: str
+        :return: SearchAlias object
+        :rtype: ~azure.search.documents.indexes.models.SearchAlias
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sample_index_crud_operations.py
+                :start-after: [START get_index]
+                :end-before: [END get_index]
+                :language: python
+                :dedent: 4
+                :caption: Get an index.
+        """
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+        result = self._client.aliases.get(name, **kwargs)
+        return result
+
+    @distributed_trace
+    def delete_alias(self, alias: Union[str, SearchAlias], **kwargs: Any) -> None:
+        # type: (Union[str, SearchAlias], **Any) -> None
+        """Deletes a search alias and its associated mapping to an index. This operation is permanent,
+        with no recovery option. The mapped index is untouched by this operation
+
+        :param alias: The alias to retrieve.
```

Comment:
This docstring is wrong. It should be the "The alias to delete." and given that we are using str/SearchAlias as a convenience, it might be better to state "The alias name or object to delete."